vcluster/_fragments/patches.mdx

```diff
@@ -3,52 +3,95 @@ import CodeBlock from '@theme/CodeBlock';
-* `context.vcluster.namespace`: Namespace of the virtual cluster
-* `context.vcluster.config`: Config of the virtual cluster, basically `vcluster.yaml` merged with the defaults
-* `context.hostObject`: Host object (can be null if not available)
-* `context.virtualObject`: Virtual object (can be null if not available)
-* `context.path`: The matched path on the object, useful when using wildcard path selectors (*)
+In the example:
 
-### Reference patches
+  - `spec.containers[*].name` targets the `name` field of all containers within a Pod. The `*` wildcard applies the patch to each container individually.
+  - `"'my-prefix-' + value"` defines a JavaScript expression that prepends `"my-prefix-"` to each container name during syncing.
+
+:::note Reverse sync
+You can use the `reverseExpression` field to define how to revert changes when syncing from the host cluster back to the virtual cluster.
+For example, add `reverseExpression: {"value.slice('my-prefix'.length)"}` to `vcluster.yaml` to remove the `"my-prefix-"` prefix when syncing back from the host cluster to the virtual cluster in the previous example.
+:::
+
+#### Context variable
+
+The context variable is an object supported in JavaScript expression patches, that provides access to virtual cluster data during syncing. The context object includes the following properties:
+
+* `context.vcluster.name`: Name of the virtual cluster.
+* `context.vcluster.namespace`: Namespace of the virtual cluster.
+* `context.vcluster.config`: Configuration of the virtual cluster, basically `vcluster.yaml` merged with the defaults.
+* `context.hostObject`: Host object (`null` if not available).
+* `context.virtualObject`: Virtual object (`null` if not available).
+* `context.path`: Matched path on the object, useful when using wildcard path selectors (`*`).
+
+### Reference patch
+
+A reference patch allows you to link a field in one resource to another resource. During syncing, vCluster rewrites the reference and imports the linked resource from a host cluster into its corresponding virtual cluster, if the resource exists.
+You can use reference patches to share resources, such as `Secrets` or `ConfigMaps`, between clusters without manually recreating or duplicating them.
+
+For example, if the host cluster contains a secret named "my-example-secret" vCluster automatically imports it into the virtual cluster. 
+Workloads in the virtual cluster can then use the secret without manual syncing.
+
+You can sync between the virtual cluster and the host cluster by mapping `spec.secretName` to a secret in the host cluster:
 
-A reference patch can be used to have a specific field of one resource point to a different resource that should get rewritten. vCluster automatically imports the referenced resource to the virtual cluster if it can find it in the host cluster. For example:
 <CodeBlock language="yaml">
-{`sync:
+{`
+sync:
   toHost:
-    ${props.resource}:
```

Comment:
Same scenario here that you need to re-use the existing codeblock that has props reference cause it spans multiple resources